### PR TITLE
INFRA-1176: auth_proxy: update login page for FB user

### PR DESCRIPTION
**Pull request by oiyana**
@yellottyellott @mccutchen Please review!

Here are three changes to the login page: 

**1. LOGIN PAGE: Updated the copy and added password reset message.**
updated-login

**2. (NEW) Add a instruction page for community users w FB login to rest the password**
instruction

**3. CSS: Add declarations to make the font rendered better**

### CODE REVIEW

**Comment by yellottyellott** [comment]
Awesome, thanks Jac! I'll get this into the auth proxy soon.

**Comment by yellottyellott** [comment]
@buzzfeed/platform-infra Take a look. The css and images are base64 encoded and inlined straight in the template. :(

**Comment by yellottyellott** [comment]
Some screenshots: 

screen shot 2016-05-11 at 9 40 48 am
screen shot 2016-05-11 at 9 41 08 am
screen shot 2016-05-11 at 9 41 19 am

Everything important is in `templates.go`, which is too big to show in the PR diff.

**Comment by mccutchen** [comment]
(scared to look)

**Comment by mccutchen** [comment]
Ah, cool, looks good!  @yellottyellott would you mind making sure that this page renders correctly in Chrome, Firefox, and Safari?  I think it should, just want to check that gigantic base64 encoded images don't give any of those browsers trouble!

**Comment by oiyana** [comment]
lol.
@mccutchen good call for checking the different browsers

@yellottyellott 
The screenshots looks good.
Let me and @cameronpaulsmith when this is ready to test on stage.

**Comment by yellottyellott** [comment]
Looks good in Safari and Firefox as well! 
I'll add a quick test to check this route and squash it all up.

File: oauthproxy_test.go [hunk 1]
```diff
@@ -329,6 +329,21 @@ func TestSignInPageDirectAccessRedirectsToRoot(t *testing.T) {
 	}
 }
 
+func TestForgotPasswordPage(t *testing.T) {
```
Comment (yellottyellott) [inline oauthproxy_test.go]: Test added here.

**Comment by mccutchen** [comment]
LGTM @yellottyellott!  🔨  🕙

**Comment by yellottyellott** [comment]
🔨 'd

**Comment by mccutchen** [comment]
👍  nice

File: static/sign_in.html [hunk 2]
```diff
@@ -40,5 +41,10 @@ <h2 class="bold xs-text-center">Log in to BuzzFeed</h2>
       <p class="text-gray--lightest text-6 xs-text-center">Secured with <a href="https://github.com/buzzfeed/auth_proxy#auth_proxy">Auth Proxy</a> version {{.Version}}</p>
        -->
     </section>
+    <section class="footnote xs-col-4 xs-py4 xs-mx-auto text-4">
```
Comment (yellottyellott) [inline static/sign_in.html]: Doh! Should we only show this when showing the password form?

Comment (oiyana) [inline static/sign_in.html]: Yes, the footnote should only show as part of the password form.

**Pull request merged**